pennylane/transforms/mitigate.py

```diff
@@ -208,8 +214,7 @@ def qfunc(op):
 
     # Generate base_circuit without measurements
     # Treat all circuits as lists of operations, build new tape in the end
-
-    base_ops = circuit.expand().copy(copy_operations=True).operations
+    base_ops = circuit.copy(copy_operations=True).operations
```

Comment:
Do we still need to copy the operations?

Reply:
Not sure why we were copying them previously as well. 🤔  I can try removing it.